std-bits/table/src/main/java/org/enso/table/formatting/TimeFormatter.java

```diff
@@ -26,6 +33,6 @@ public String format(Object value) {
 
   @Override
   public boolean canFormat(Object value) {
-    return value instanceof LocalTime;
+    return value instanceof LocalTime || (value instanceof Value v && !v.isDate() && v.isTime());
```

Comment:
Looks familiar. I played with something similar.

Reply:
Yeah, I don't see other way around it.